public/app/plugins/panel/timeseries/NullsThresholdInput.tsx

```diff
@@ -8,15 +8,27 @@ export enum InputPrefix {
   GreaterThan = 'greaterthan',
 }
 
-type Props = { value: number; onChange: (value?: number | boolean | undefined) => void; inputPrefix?: InputPrefix };
+type Props = {
+  value: number;
+  onChange: (value?: number | boolean | undefined) => void;
+  inputPrefix?: InputPrefix;
+  isTime: boolean;
+};
 
-export const NullsThresholdInput = ({ value, onChange, inputPrefix }: Props) => {
-  const formattedTime = rangeUtil.secondsToHms(value / 1000);
+export const NullsThresholdInput = ({ value, onChange, inputPrefix, isTime }: Props) => {
+  let defaultValue = rangeUtil.secondsToHms(value / 1000);
+  if (!isTime) {
+    defaultValue = '10';
+  }
   const checkAndUpdate = (txt: string) => {
     let val: boolean | number = false;
     if (txt) {
       try {
-        val = rangeUtil.intervalToMs(txt);
+        if (isTime && rangeUtil.isValidTimeSpan(txt)) {
+          val = rangeUtil.intervalToMs(txt);
+        } else {
+          val = Number(txt);
+        }
```

Comment:
This block determines how we handle user input for threshold - by default `isTime` is set to `true` - in the case of trend panel we are passing `false` in [module.tsx](https://github.com/grafana/grafana/pull/70616/files#diff-6740947482693d9c515c8b06c21799c065b5d10759d0a6748af5a7c78d73994dR11)